content/master/concepts/composite-resource-definitions.md

```diff
@@ -0,0 +1,808 @@
+{{<hover label="ver" line="29">}}region{{</hover>}} and 
+{{<hover label="ver" line="31">}}size{{</hover>}}.
+
+```yaml {label="ver",copy-lines="none"}
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: xdatabases.custom-api.example.org
+spec:
+  group: custom-api.example.org
+  names:
+    kind: xDatabase
+    plural: xdatabases
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        type: object
+        properties:
+          spec:
+            type: object
+            properties:
+              region:
+                type: string  
+  - name: v1
+    schema:
+      openAPIV3Schema:
+        type: object
+        properties:
+          spec:
+            type: object
+            properties:
+              region:
+                type: string 
+              size:
+                type: string            
+```
+
+{{<hint "important" >}}
+
+Changing or expanding the XRD schema requires restarting the [Crossplane
+pod]({{<ref "./pods#crossplane-pod">}}) to take effect.
+{{< /hint >}}
+
+### Enable Claims
+
+Optionally, XRDs can allow Claims to use the XRD API. 
+
+{{<hint "note" >}}
+
+Read the [Claims]({{<ref "./claims">}}) page for more
+information about Claims.
+{{</hint >}}
+
+XRDs offer Claims with a 
+{{<hover label="claim" line="10">}}claimNames{{</hover >}} object.
+
+The {{<hover label="claim" line="10">}}claimNames{{</hover >}} defines a 
+{{<hover label="claim" line="11">}}kind{{</hover >}} and 
+{{<hover label="claim" line="12">}}plural{{</hover >}} like the XRD 
+{{<hover label="claim" line="7">}}names{{</hover >}} object.   
+Also like XRD 
+{{<hover label="claim" line="7">}}names{{</hover >}}, use UpperCamelCase
+for the
+{{<hover label="claim" line="11">}}kind{{</hover >}} and lowercase for the 
+{{<hover label="claim" line="12">}}plural{{</hover >}}.
+
+The Claim 
+{{<hover label="claim" line="11">}}kind{{</hover >}} and 
+{{<hover label="claim" line="12">}}plural{{</hover >}} must be unique. They
+can't match any other Claim or other XRD 
+{{<hover label="claim" line="8">}}kind{{</hover >}}.
+
+{{<hint "tip" >}}
+Common Crossplane convention is to use 
+{{<hover label="claim" line="10">}}claimNames{{</hover >}} that match the XRD 
+{{<hover label="claim" line="7">}}names{{</hover >}}, but without the beginning
+"x."
+{{</hint >}}
+
+```yaml {label="claim",copy-lines="none"}
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: xdatabases.custom-api.example.org
+spec:
+  group: custom-api.example.org
+  names:
+    kind: xDatabase
+    plural: xdatabases
+  claimNames:
+    kind: Database
+    plural: databases
+  versions:
+  # Removed for brevity
+```
+
+{{<hint "important" >}}
+You can't change the `claimNames` after they're defined. You must delete and
+recreate the XRD to change the `claimNames`.
+{{</hint >}}
+
+### Manage connection secrets
```

Comment:
From what I've seen so far, it looks like we are breaking up the steps/details of how to specify connection information into each specific component (XRDs, XRs, Compositions, etc.)

What I really like about https://github.com/crossplane/docs/pull/485 is that it has an overall summary of the entire flow across all components, which gives the reader a single place to understand how it all comes together.

Do we have that single summary in this PR? of if so, can we make it more discoverable/findable? 🤔

Reply:
I think there is a TON of value in what you're spelling out, but I think it's a "thing to do next" item.

By breaking up all of the concepts like this mega PR does, we lose that "all in one place" reference. I 100% believe we need to create a more comprehensive deployment example that can pull a thread through all of the components.

Reply:
do you want to track it with a new issue then? I think that after the effort to pull all the steps together to get connection details working in https://docs.crossplane.io/v1.12/concepts/composition/#composite-resource-connection-secrets, it's a bit of a step backwards to lose that top to bottom summary, so at least having a tracking issue would be my request 🙇

Reply:
KB added- 
https://deploy-preview-475--crossplane.netlify.app/knowledge-base/guides/connection-details/